sessions/react-state-3/journal-app-favorite-button-shared-state/src/components/EntriesSection/index.js

```diff
@@ -0,0 +1,30 @@
+import "./EntriesSection.css";
+import Divider from "../Divider";
+import Entry from "../Entry";
+import Tabs from "../Tabs";
+import Tab from "../Tab";
+import Badge from "../Badge";
+import { Fragment } from "react";
+
+export default function EntriesSection({ entries }) {
+  return (
+    <section className="entries-section">
+      <Tabs>
+        <Tab active>
+          All Entries <Badge active>3</Badge>
+        </Tab>
+        <Tab>
+          Favorites <Badge>1</Badge>
+        </Tab>
+      </Tabs>
+      <div className="entries-section__entries">
+        {entries.map((entry, index) => (
+          <Fragment key={entry.id}>
+            <Entry date={entry.date} motto={entry.motto} notes={entry.notes} />
+            {index < entries.length - 1 ? <Divider /> : null}
```

Comment:
is this maybe easier to understand?

```suggestion
            {index !== 0 ? <Divider /> : null}
            <Entry date={entry.date} motto={entry.motto} notes={entry.notes} />
```

Reply:
Good idea! I decided to go with `{index > 0 ? <Divider /> : null}` which feels a bite more intuitive